### PR TITLE
Updated Instructions

### PR DESCRIPTION
**Pull request by iogburu**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [X] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [X] Tested changes locally.
- [ ] Closes currently open issue (`Closes #XXXX`): Closes 
#### Description

Updated Instructions for "Use Responsive Design with Bootstrap Fluid Containers" section in order to inform users who may jump around in the tutorials where they can learn how to build the Cat Photo App.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/bootstrap.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
       "id": "bad87fee1348bd9acde08712",
       "title": "Use Responsive Design with Bootstrap Fluid Containers",
       "description": [
-        "Now let's go back to our Cat Photo App. This time, we'll style it using the popular Bootstrap responsive CSS framework.",
+        "In the HTML5 and CSS section of FreeCodeCamp we built a Cat Photo App. Now let's go back to our Cat Photo App. This time, we'll style it using the popular Bootstrap responsive CSS framework.",
```
Comment (theflametrooper) [inline seed/challenges/01-front-end-development-certification/bootstrap.json]: Repetitive use of `Cat Photo App`, "we built a Cat Photo App. Now let's go back to our Cat Photo App."

Comment (iogburu) [inline seed/challenges/01-front-end-development-certification/bootstrap.json]: This update was made, as not everyone will have performed the previous activity. The line was added to ensure participants understand where the previous code came from.

Comment (theflametrooper) [inline seed/challenges/01-front-end-development-certification/bootstrap.json]: Try and reword the sentences so you don't use the word `CatPhotoApp` more than once.

**Comment by iogburu** [comment]
@theflametrooper Cat Photo App has been redundancy has been removed.

**Comment by theflametrooper** [comment]
@iogburu Please squash your commits

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @iogburu and @theflametrooper .

**Pull request merged**